lib/europeana/api/requestable.rb

```diff
@@ -0,0 +1,92 @@
+require 'active_support/core_ext/hash/indifferent_access'
+
+module Europeana
+  module API
+    ##
+    # Mixin for classes that need to make Europeana API requests
+    #
+    # Class needs to implement {#request_url}
+    module Requestable
+      ##
+      # Request-specific params, to be overriden in including class
+      #
+      # @return [Hash]
+      def params
+        {}
+      end
+
+      ##
+      # Query params with API key added
+      #
+      # @return [Hash]
+      def params_with_authentication
+        return params if params.key?(:wskey) && params[:wskey].present?
+        unless Europeana::API.api_key.present?
+          fail Europeana::API::Errors::MissingAPIKeyError
+        end
+        params.merge(wskey: Europeana::API.api_key)
+      end
+
+      ##
+      # Execute the API request
+      #
+      # @param options [Hash] Options sent on to {#request_uri} and {#parse_response}
+      # @return (see #parse_response)
+      # @raise [Europeana::Errors::ResponseError] if API response could not be
+      #   parsed as JSON
+      def execute_request(options = {})
+        response = Request.new(request_uri(options)).execute
+        parse_response(response, options)
+      rescue JSON::ParserError
+        raise Errors::ResponseError
+      end
+
+      ##
+      # Parses a JSON response from the API
+      #
+      # @param response (see Net::HTTP#request)
+      # @param options [Hash] Options used by including module implementation
+      # @return [HashWithIndifferentAccess] Parsed body of API response
+      def parse_response(response, options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.